pages/en/download/releases.md

```diff
@@ -1,11 +0,0 @@
----
-layout: download-releases.hbs
-title: Previous Releases
-modules: '<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier versions, but is now represented as an integer.'
----
-
-### io.js & Node.js
-
-Releases 1.x through 3.x were called "io.js" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases.
```

Comment:
This has been removed in the transition to `pages/en/about/previous-releases.md`? 

I see the iojs releases are missing from the current https://nodejs.org/en/download/releases. They were there on the old version of nodejs.org, e.g. http://web.archive.org/web/20221228040231/https://nodejs.org/en/download/releases/

Reply:
Yes, the intent is to not show the iojs releases anymore, and focus only on showing Node.js releases. (But this was like this since the Next.js migration)

I've just now updated the copies.